docs/tips-and-tricks.md

```diff
@@ -244,6 +244,32 @@ To connect to the jean-zay cluster you will then just need to do `ssh jz`.
 For good practices about SSH keys you can have a look at:
 http://www.idris.fr/faqs/ssh_keys.html
 
+### Automatic synchronization with your local machine
+
+The following script allows you to automatically synchronize a local directory and have an exact copy of it on jean-zay.
+For the script to run smoothly, make sure the directory is lightweight, e.g. a directory containing code. On your local machine, create a file `sync_jz.sh` with
+
+```bash
+#!/bin/bash
+
+source_path="/your/local/directory"
+target_path="jz-username@jean-zay.idris.fr:/your/jean-zay/directory"
+
+while inotifywait -r -e modify,create,delete $source_path
+do
+    rsync -azh $source_path $target_path \
+          --progress \
+          --delete --force \
+          --exclude=".git"
+done
+```
+
+To make the script executable run `chmod +x sync_jz.sh` then run `./sync_jz.sh`.
```

Comment:
I think it would be nice to specify here what will happen (I am not familiar at all with `inotify`).

For example: will this process continue after I restart my local machine or do I have to run this script at every restart?
will this run in the background or do I have to keep my terminal session open locally?

Reply:
Hi @zaccharieramzi , glad that this is useful!
It needs to be launched after each boot of your local machine. It does not run in the background and need to be kept in a terminal session (I usually use tmux and let it there). It can probably be made to be run in the background. However, it's nice to have a terminal session where you can see the progress of sync or the last file synced as it allows to spot problems sometimes. I will be more explicit in the doc.

Reply:
Yes of course, I find it very nice indeed even if it needs to be relaunched, I just wanted to be sure of the use.
Thanks for clarifying.